src/Provider/Photon/Tests/PhotonTest.php

```diff
@@ -135,6 +135,25 @@ public function testGeocodeQueryWithMultipleOsmTagFilter(): void
         $this->assertGreaterThan(0, $countGalleries);
     }
 
+    public function testGeocodeQueryWithLatLon(): void
+    {
+        $provider = Photon::withKomootServer($this->getHttpClient());
+
+        $query = GeocodeQuery::create('Paris')->withLimit(1);
+        $results = $provider->geocodeQuery($query);
+        $this->assertInstanceOf('Geocoder\Model\AddressCollection', $results);
```

Comment:
```suggestion
        $this->assertInstanceOf(AddressCollection::class, $results);
```

Don't forget to import `AddressCollection`.

Reply:
There are multiple assertions like this in that test file. Is it ok to update them all in this PR, or should I change them only in new test?

Reply:
Only focus on the ones in the Photon provider to keep the PR clear and simple!
If you have the energy and time to do it everywhere, that would be awesome but I would do it in a separate PR. Thanks.

Reply:
Resolved in this test. Leaving other cases for another PR.